examples/html-kitchen-sink/stories/addon-docs.stories.mdx

```diff
@@ -0,0 +1,20 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+
+# Storybook Docs for HTML
+
+<Meta title="Addons|Docs" />
+
+## Story definition
+
+Hallelujah! HTML is working out of the box without modification.
+How you like them apples?!
+
+<Story name="heading" height="100px">
+  {'<h1>Hello World</h1>'}
```

Comment:
I wonder whether we can make JSX work here

Reply:
We probably can. To start with I want to have the `<Story>` contents have a 1-to-1 correspondence to the return function from the story function, for simplicity's sake. In 6.0 (possibly in conjunction with some subset of multiframework and monoconfig work) we can rethink this on a framework-by-framework basis.

Also in the meantime, MDX may evolve to support some of this stuff natively. I know Vue support is in alpha, for example.

Reply:
Vue templates are valid HTML strings so maybe it would be enough